### DISCUSSION TITLE
Multiple RequestUrlHost values for tenant

### DISCUSSION THREAD

**Discussion by MikeKry** [OrchardCMS/OrchardCore#9915] [Q&A] [answered]
Hi, 
I would like to have mutliple RequestUrlHost values for tenant, so I can for example set it to "www.project.com,project.com,project-ab.com" and it would work on all three hosts

Is this possible?

**Comment by deanmarcussen** [comment, marked as answer]
Multiple urls is not a great idea for seo / search engines.

I would suggest using rewrites with whatever platform you are hosting with to redirect to the preferred host.

**Reply by MikeKry** [reply]
Agreed, but when you have mutliple language versions, you might want to have different domains for each language, that is not possible right now.

**Reply by MikeKry** [reply]
for example:
english -> myenglissite.com
french -> myfrenchsite.fr

instead of mysite.com/en and mysite.com/fr

**Reply by deanmarcussen** [reply]
So from a language pov that makes more sense than your original suggestion.

Suggest having a look at the matching options that are available already in the `IRunningShellTable` as that should suit your scenario.

What we don`t have currently is a culture selector based on prefixes, but that should not be a dificult feature to add. People will want it at some point.

**Reply by MikeKry** [reply]
thanks, seems it is ok for me to use mutliple hosts like this:
"RequestUrlHost": "localhost:44301, www.localhost:44301" and then handle cultures myself as I have headless app anyway
